projects/addon-doc/components/example/example.component.ts

```diff
@@ -56,6 +57,8 @@ import {TuiDocExampleGetTabsPipe} from './example-get-tabs.pipe';
         TuiMapperPipe,
         NgTemplateOutlet,
         TuiDocCode,
+        TuiLink,
+        RouterLinkActive,
```

Comment:
Is it used? I see you did manual check for active link, would be nice to use Angular directive for that.

Reply:
fixed